mirgecom/euler.py

```diff
@@ -199,6 +199,18 @@ def inviscid_flux(discr, eos, q):
     The inviscid fluxes are
     $(\rho\vec{V},(\rho{E}+p)\vec{V},\rho(\vec{V}\otimes\vec{V})
     +p\mathbf{I}, \rho{Y_s}\vec{V})$
+
+    Note
+    ----
+    The fluxes are returned as a 2D object array with shape:
+    ``(num_equations, ndim)``.  Each entry in the
+    flux array is a :class:`~meshmode.dof_array.DOFArray`.  This
+    form and shape for the flux data is required by the built-in
+    state data handling mechanism in :mod:`mirgecom.euler`. That
+    mechanism is used by at least
+    :class:`mirgecom.euler.ConservedVars`, and
+    :func:`mirgecom.euler.join_conserved`, and
+    :func:`mirgecom.euler.split_conserved`.
```

Comment:
```suggestion
    .. note::

	      The fluxes are returned as a 2D object array with shape:
	      ``(num_equations, ndim)``.  Each entry in the
	      flux array is a :class:`~meshmode.dof_array.DOFArray`.  This
	      form and shape for the flux data is required by the built-in
	      state data handling mechanism in :mod:`mirgecom.euler`. That
	      mechanism is used by at least
	      :class:`mirgecom.euler.ConservedVars`, and
	      :func:`mirgecom.euler.join_conserved`, and
	      :func:`mirgecom.euler.split_conserved`.
```

Reply:
The indentation should not be tabs here right?

Reply:
Ugh, not sure.  I failed to see this suggestion and then merged without resolving.  Is there any way to recover gracefully?

Reply:
#223 🤦

Reply:
> The indentation should not be tabs here right?

Yup, very true.

Reply:
I fixed the formatting in #223

Reply:
#224